stdlib/2and3/modulefinder.pyi

```diff
@@ -29,7 +29,7 @@ class ModuleFinder:
     excludes: Container[str]  # undocumented
     replace_paths: Sequence[Tuple[str, str]]  # undocumented
 
-    def __init__(self, path: Optional[List[str]] = ..., debug: int = ..., excludes: Container[str] = ..., replace_paths: Sequence[Tuple[str, str]] = ...) -> None: ...
+    def __init__(self, path: Optional[List[str]] = ..., debug: int = ..., excludes: Optional[Container[str]] = ..., replace_paths: Optional[Sequence[Tuple[str, str]]] = ...) -> None: ...
```

Comment:
Why is this `Optional`? On 3.7 the default is `[]` and code inside the class does `x in self.excludes` without checking for None.

Reply:
The default value is None on 3.8 and it explicitly checks for None. I'll version-gate the change.